jenkins/nginx/nginx.conf

```diff
@@ -0,0 +1,104 @@
+worker_processes 1;
+
+daemon off;
+
+events {
+    worker_connections 1024;
+    use epoll;
+}
+
+http {
+
+    # server {
```

Comment:
Commented code = unused code. Remove or provide further comments why we shouldn't delete this